README.md

```diff
@@ -1,32 +1,197 @@
+0.77 (M2) [issue#8](https://github.com/ivadomed/model-spinal-rootlets/issues/8).
+
+The previous dataset (D2) has been modified by changing voxel values according to the spinal level (D3). An initial
+training of 5 folds with 1000 epochs has been done. After reviewing the results and manually correcting the dataset (
+sct_maths -d 1 used to help labeling this time), a new training of 5 folds with 2000 epochs is in progress.
+
+### C) Reproduce
+- Find labeled images 
+- Create different dataset 
+- Run training 
+- Run inference 
+
+In the next section all the instruction to reproduce the label file used in the final dataset will be described. Hoever label file are also available here (give to dataset with root_label)
+
+#introduce script to convert BIDS->nnUNet and nnUNet->BIDS
+#explain how to label on FSLeyes 
+
+#### i) Reproduce D1a, M1a and D1b, M1b
+Clone the original dataset D0a
+```
+git clone https://github.com/OpenNeuroDatasets/ds004507.git
+```
+#Explain session
+#Explain why don't take HeadDown 
+Linked to [issue#5](https://github.com/ivadomed/model-spinal-rootlets/issues/5)
+
+With Fsleyes manually segment the following file: 
+<details>
+<summary>12 first images to label</summary>
+```
+sub-002_ses-headNormal_T2w_root-manual.nii.gz	
+sub-002_ses-headUp_T2w_root-manual.nii.gz	
+sub-003_ses-headNormal_T2w_root-manual.nii.gz
+sub-003_ses-headUp_T2w_root-manual.nii.gz
+sub-004_ses-headNormal_T2w_root-manual.nii.gz	
+sub-004_ses-headUp_T2w_root-manual.nii.gz
+sub-005_ses-headNormal_T2w_root-manual.nii.gz
+sub-005_ses-headUp_T2w_root-manual.nii.gz
+sub-006_ses-headNormal_T2w_root-manual.nii.gz
+sub-006_ses-headUp_T2w_root-manual.nii.gz
+sub-007_ses-headNormal_T2w_root-manual.nii.gz
+sub-007_ses-headUp_T2w_root-manual.nii.gz
+```
+</details>
+
+> You can use the `json_write.py` script to add the json file according to the .nii.gz file created
+
+Now convert this Bids dataset to a nnUNet dataset `link to command script`. 
+This is the D1a dataset (100% train image no test image)
+
+Add the 
+<details>
+<summary>dataset.json</summary>
+```
+{
+    "channel_names": {
+        "0": "T2w"
+    },
+    "labels": {
+        "background": 0,
+        "label": 1
+    },
+    "numTraining": 12,
+    "file_ending": ".nii.gz",
+    "overwrite_image_reader_writer": "SimpleITKIO"
+}
+```
+</details>
+
+Train model D1a with : `CUDA_VISIBLE_DEVICES=XXX nnUNetv2_train DATASETID 3d_fullres 0`
+
+> You can stop when the progress.png reach a plateau (approx 250)
+
+#Add create a dataset with all D0a in imagesTs
+
+Predict all the segmentation of D0a dataset with the model M1a with `nnUNetv2_predict -i PATH_TO:imagesTs -o PATH_TO:Out_directory -d DATASETID -c 3d_fullres --save_probabilities -chk checkpoint_best.pth`
+
+Manually review the predicted label. In my case I have dropped `sub-006-Normal` `sub-009-Normal` I was not satisfy with the quality. 
```

Comment:
```suggestion
Manually review the predicted labels. 
Note: subjects `sub-006-Normal` and `sub-009-Normal` have been dropped since they did not satisfy the quality. 
```